run.sh

```diff
@@ -758,6 +758,18 @@ run_dascript() {
     check_output "related_posts_dascript.json"
 }
 
+run_lobster() {
+    echo "Running Lobster" &&
+        cd ./lobster &&
+        if [ $HYPER == 1 ]; then
+            capture "Lobster" hyperfine -r $slow_lang_runs -w $warmup --show-output "lobster related.lobster"
+        else
+            command ${time} -f '%es %Mk' lobster related.lobster
```

Comment:
This is the default JIT run. You'll want to also time a compiled run.

Reply:
That's `lobster --cpp related.lobster` right?

Reply:
Yup.. that generates a .cpp, which will need to be compiled and linked with the rest of Lobster: https://aardappel.github.io/lobster/implementation.html

Reply:
Can't get it to output the cpp file: https://github.com/aardappel/lobster/issues/273